example/05_b_overlay_linestring_polygon_example.cpp

```diff
@@ -72,8 +72,8 @@ int main(void)
     std::vector<turn_info> turns;
     bg::detail::get_turns::no_interrupt_policy policy;
     bg::detail::no_rescale_policy rescale_policy;
-    bg::strategy::intersection::services::default_strategy<typename bg::cs_tag<point_2d>::type>::type intersection_strategy;
-    bg::get_turns<false, false, bg::detail::overlay::assign_null_policy>(ls, p, intersection_strategy, rescale_policy, turns, policy);
+    typename bg::strategies::relate::services::default_strategy<bg::model::linestring<point_2d>, bg::model::polygon<point_2d>>::type strategy;
```

Comment:
None of the types are dependant so `typename` shouldn't be used. AFAIR some compilers may report error here. Or am I missing something?

Reply:
Indeed, will fix.

Reply:
:heavy_check_mark: